src/classic/hfp_ag.c

```diff
@@ -2859,6 +2859,16 @@ uint8_t hfp_ag_set_battery_level(int battery_level){
     return ERROR_CODE_SUCCESS;
 }
 
+uint8_t hfp_ag_set_operator_name(hci_con_handle_t acl_handle, const char* operator_name){
+
+    struct hfp_connection* connection = get_hfp_connection_context_for_acl_handle(acl_handle,HFP_ROLE_AG);
+    hfp_network_opearator_t operatorData;
+    operatorData.mode = 3;
+    operatorData.format = 0;
```

Comment:
I know it's not always respected in btstack to set things from some compile-time constants defined but could we? :) 
Therefore consider being kind and defining somewhere ( via define at the top? ) 
From some random COPS documentation online, possible COPS values here:
```
<mode>
    0 – Automatic network selection
    1 – Manual network selection
    3 – Set <format> of +COPS read command response
<format>
    0 – Long alphanumeric <oper> format. Only for <mode> 3.
    1 – Short alphanumeric <oper> format. Only for <mode> 3.
    2 – Numeric <oper> format
```

Reply:
sure, that would make it more verbose :P

Reply:
done